src/utils/api.js

```diff
@@ -94,7 +94,10 @@ const callAPI = (route, options) => {
 
       if (constants.DEBUG || global.argOpts.debug) {
         console.log(`>> ${requestOptions.method} ${requestOptions.url}`);
-        if (requestOptions.body) { console.log(`>> ${requestOptions.body}`); }
+        if (requestOptions.body) {
+          let cleanedBody = _.assign({}, JSON.parse(requestOptions.body), {zip_file: 'raw zip removed in logs'});
```

Comment:
That's correct! It comes out as a ton of text, which isn't helpful. Makes debugging really frustrating.